croncert-config.yml

```diff
@@ -1407,6 +1407,55 @@ scrapers:
         date_location: "Europe/Berlin"
         date_language: "nl_NL"
 
+  ##########
+  # Nyon
+  ##########
+  - name: Parenthese
+    url: http://www.bar-laparenthese.ch/
+    item: div.event
+    fields:
+      - name: location
+        value: La Parenthèse
+      - name: city
+        value: Nyon
+      - name: type
+        value: concert
+      - name: "sourceUrl"
+        value: "http://www.bar-laparenthese.ch/"
+      - name: title
+        type: text
+        location:
+          selector: div.event_title > h3 > a
+      - name: url
+        type: url
+        location:
+          attr: href
+          selector: div.event_title > h3 > a
+      - name: date
+        type: "date"
+        on_subpage: url
+        date_language: fr_FR
+        date_location: "Europe/Paris"
+        components:
+          - covers:
+              day: true
+              month: true
+              year: true
+            location:
+              selector: div.post > div.entry > p
+              child_index: 1
+            layout: ["January 2, 2006"]
+          - covers:
+              time: true
+            location:
+              selector: div.post > div.entry > p:nth-child(3)
+              child_index: 1
+            layout: ["15:04"]
+    filters:
```

Comment:
I removed the eventType although this information is definitely interesting. However, right now any extra field is ignored by the api that this concert data is written to. I can imagine though that we'll add extra (optional) fields to the api/database. This would have to be done in https://github.com/jakopako/event-api Probably in this specific case it'd be sth like a 'genre'.

Also note that there is the 'type' field which right now always has 'concert' as value. To be honest this is kind of superfluous right now but at some point I thought that the [event-api](https://github.com/jakopako/event-api) doesn't necessarily only manage concerts but also other types of events.

Reply:
Yeah, eventually it would be good to have other kinds of events, I think, but better one step at a time.

When I tried to do something like this in the '00s I think I went too far too fast by supporting too many event types. I also failed to see it as a scraper problem, and tried to do it with Mediawiki. So I was writing scrapers and everybody else was hand-entering things.